### PR TITLE
Lra 500 lsa ride share cars index page

### PR DESCRIPTION
**Pull request by britaumich**
This is the Cars index page. All buttons go to the right pages but those pages will be designed in different pull requests. The design was done by Maria.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -69,4 +69,16 @@ def render_flash_stream
     turbo_stream.update "flash", partial: "layouts/notification"
   end
 
+  def vehicle_reports(car)
+    if @vehicle_reports = VehicleReport.where(reservation_id: car.reservations.ids).order(:updated_at).present?
```
Comment (rsmoke) [inline app/helpers/application_helper.rb:73]: Another example of using tell don't ask. Will definitely benefit us going forward

File: app/policies/vehicle_report_policy.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    @user = user
+    @record = record
+  end
+
+  def index?
+    user_in_access_group?
+    # true
+  end
+
+  # def show?
```
Comment (rsmoke) [inline app/policies/vehicle_report_policy.rb:16]: We're not using these comments. Let's get rid of them

**Review by rsmoke — CHANGES_REQUESTED** [review]
Good work on the styles.
Getting rid of the conditional and using the tell don't ask again will help you.
Let's get rid of the comments if you're not going to use them and not wait for a code review later to clean it up

File: app/policies/vehicle_report_policy.rb (hunk 2 above)
Comment (britaumich) [reply app/policies/vehicle_report_policy.rb]: comments removed

**Comment by britaumich** [comment]
I changed the way to show a link to the car's vehicle reports. 
The Car model has a method to get all vehicle reports for a particular car: Car.first.vehicle_reports_ids
The VehicleReport model has a scope to display some reports (if ids are present) or all reports.
The vehicle_reports_controller's index display reports according to that scope.
The car's index view displays the link to vehicle reports only if there are reports for that car.

**Pull request merged**